crates/mfem/Cargo.toml

```diff
@@ -9,13 +9,15 @@ edition = "2021"
 repository = "https://github.com/mkovaxx/mfem-rs"
 
 [dependencies]
-cxx = "1.0.137"
+cxx = "1.0.157"
+autocxx = "0.28"
 mfem-sys = { version = "0.2.0", path = "../mfem-sys" }
-thiserror = "1.0.59"
+paste = "1.0"
+# aquamarine = "0.6.0"
```

Comment:
nit

Reply:
Fixed (rewriting history).

Reply:
Remark: `aquamarine` may be useful to draw dependency graphs as they have in the MFEM doc.